uber/custom_tags.py

```diff
@@ -376,6 +401,16 @@ def int_options(minval, maxval, default=1):
     return safe_string('\n'.join(results))
 
 
+@JinjaEnv.jinja_export
+def format_location(location, separator='<br>', spacer='above', text_class='text-nowrap'):
```

Comment:
This seems to be doing some really strange stuff. If you don't mind, can you explain what's up here? It's not urgent, so if you need to wait till after Super that's fine.

Reply:
Yeah, I'll add a comment explaining what's going on!

Reply:
See here: https://github.com/magfest/ubersystem/pull/3056